### DISCUSSION TITLE
Monitor => Наставник, Монитор, Наблюдатель или?

### DISCUSSION THREAD

**Discussion by tigran123** [tigran123/ud#5] [Идеи и вопросы]
Сегодня Ирина Чернова написала мне по поводу недавнего изменения в русском переводе Откровения: слово "Monitor", относящееся к Настройщику Мыслей (духу Отца), ранее переведённое как "Наставник" я изменил на "Монитор". Ирина возражает против этого изменения, ибо перевод "Монитор" создаёт впечатление "скрытой камеры наблюдения", установленной в нас "свыше" и ведущей за нами наблюдения независимо от нашей воли.

Я ответил следующее: Перевод "Наставник" мне не понравился из-за того, что я узнал в нём стандартную ошибку перевода Фонда, а именно, в трудных ситуациях переводить не само слово, а другое, близкое к нему ПО ЗВУЧАНИЮ. В данном случае, похоже, что переводчик заменил "Monitor" => "Mentor" и получился "Наставник". Здесь смысл "monitor" от "monition" (как в "premonition") — "наблюдение". Можно перевести как "Наблюдатель". Да, Настройщик, конечно, наставляет, т.е. указывает правильное направление к Отцу, это верно. И остальные все переводы перевели как "Наставник". Ну, хорошо, давай подумаем — если ничего лучше не придумаем, я верну вариант "Наставник".

Вот мы и решили узнать мнение остальных участников проекта: как лучше перевести "Monitor"? (spirit Monitor, Mystery Monitor и т.д.)

**Comment by irinachernov** [comment]
Есть ещё слово "Куратор". Как такой вариант? Этот вариант был выдан в поисковике для обоих слов: "Наставник" и "Наблюдатель".

**Comment by tigran123** [comment]
В конце концов я остановился на слове "Монитор", но добавил следующую сноску к 1:2.3:

Английское слово Monitor встречается по всему Откровению во фразах spirit Monitor и Mystery Monitor. В 92:3.7 Настройщик Мыслей Мелхиседека называется «монитором времени и наставником плоти» (англ. the monitor of time and the mentor of the flesh). Для однородности перевода я выбрал термин _Монитор,_ однако следует остерегаться представления о духе моего Отца, как о пассивном наблюдателе-_мониторе,_ а также избегать иллюзорной идеи активного глашатая-_наставника,_ за голос которого многие, увы, слишком часто принимают отголоски собственного подсознания.
